docs/testplans/study-viewer/study-viewer.md

```diff
@@ -0,0 +1,32 @@
+# Study Viewer
+
+## Overview
+Dataset Browser allows the user visiting the commons to discover if the dataset they need is available on the commons. The user doesn't need to have full access to look for the dataset they are interested in. Users can request access to data they don't have access to.
+
+It enables to the user to check :
+1. if Gen3 has the data they need
+2. if the data is sufficient enough for research 
+
+## Technical Document
+[Dataset browser/ Study Viewer](https://docs.google.com/document/d/1BLbLX4GEViJfcWFNDNN723KmC2XMHH-SWFMM_cPg6wk/)
+
+[Access Request Queue](https://docs.google.com/document/d/1h5ZLYXb_wi2a2H3sfXrRcY41KQ0SjQF3DBtdpt2oLxE/)
+
+## Workflow
+
+1. The users visits the commons 
+2. Navigate to the Dataset Browser/ Study Viewer Tab on the navigation bar
+3. The user searches for the dataset that they need and clicks 'Show Details'
+4. if they have access, 'Download' button is displayed
+5. if they do not have access, 'Request Access' button is displayed
+6. Click on download button, if file -> file is downloaded (first iteration)
+
+## Tests Scenarios
```

Comment:
sorry, just realized we're missing something else - the study viewer has 2 pages so i expect you'd need a test case to navigate from one to the other